src/script_chainer/utils/process_utils.py

```
@@ -32,3 +34,33 @@ def graceful_kill_popen(proc: subprocess.Popen, timeout: float = 3) -> None:
+        title: 终端窗口标题。
+
+    Returns:
+        启动的 Popen 对象。
+    """
+    if sys.platform == 'win32' and shutil.which('wt'):
+        wt_cmd = ['wt']
+        if title:
+            wt_cmd.extend(['--title', title])
+        wt_cmd.append('--')
+        wt_cmd.extend(command)
+        return subprocess.Popen(wt_cmd, cwd=cwd)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)